components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss

```diff
@@ -210,11 +210,42 @@ chef-alert {
   }
 
   .healthcheck-content {
+    position: relative;
     border-radius: 0 0 2px 2px;
     background-color: var(--chef-primary-dark);
     color: var(--chef-white);
-    padding: 8px;
+    padding: 8px 8px 40px 8px;
     overflow-wrap: break-word;
+    max-height: 94px;
+    overflow: hidden;
+    transition: .25s max-height;
```

Comment:
I think i want to add a specific bezier curve here for looks - the reason we're transitioning to specific max-heights is so that we can animate it.  This css animation is cheap and lets us slide the text open and closed for a less jarring effect.